src/markup.c

```diff
@@ -28,6 +25,35 @@ char *markup_quote(char *str)
         return str;
 }
 
+static
+char *markup_unquote(char *str)
+{
+        if (str == NULL) {
```

Comment:
Alright, update the static functions then and I'll investigate later the possibility of doing the same thing in `markup_strip` and `markup_transform`.